Yosemite/Yosemite/Stores/FeatureFlagStore.swift

```diff
@@ -0,0 +1,67 @@
+import Networking
+import Storage
+
+public final class FeatureFlagStore: Store {
+    private let remote: FeatureFlagRemoteProtocol
+
+    init(dispatcher: Dispatcher,
+         storageManager: StorageManagerType,
+         network: Network,
+         remote: FeatureFlagRemoteProtocol) {
+        self.remote = remote
+        super.init(dispatcher: dispatcher, storageManager: storageManager, network: network)
+    }
+
+    public override convenience init(dispatcher: Dispatcher,
+                                     storageManager: StorageManagerType,
+                                     network: Network) {
+        self.init(dispatcher: dispatcher,
+                  storageManager: storageManager,
+                  network: network,
+                  remote: FeatureFlagRemote(network: network))
+    }
+
+    // MARK: - Actions
+
+    /// Registers for supported Actions.
+    ///
+    override public func registerSupportedActions(in dispatcher: Dispatcher) {
+        dispatcher.register(processor: self, for: FeatureFlagAction.self)
+    }
+
+    /// Receives and executes Actions.
+    /// - Parameters:
+    ///   - action: An action to handle. Must be a `FeatureFlagAction`
+    ///
+    override public func onAction(_ action: Action) {
+        guard let action = action as? FeatureFlagAction else {
+            assertionFailure("StoreOnboardingTasksStore received an unsupported action")
```

Comment:
```suggestion
            assertionFailure("FeatureFlagStore received an unsupported action")
```

Reply:
Good catch, fixed in https://github.com/woocommerce/woocommerce-ios/pull/9700/commits/354d1051f6b006f300976eb4d9cb5f02a6502910